setup.py

```diff
@@ -40,7 +40,7 @@
                       'boto >= 2.0b4',
                       'bunch >= 1.0.0',
                       'configobj',
-                      'six',
+                      'six >= 1.9', # python-openstackclient won't work properly with less
```

Comment:
I think we're fine requiring a newer six here; if this was a ``<=`` I'd be opposed.